spec/src/main/asciidoc/WebServicesMetadata.adoc

```diff
@@ -943,2259 +807,1399 @@ header, and the mode is OUT.
-https://jax-ws.dev.java.net/issues/show_bug.cgi?id=710]#. The
-requirement to specify a name for "document" style, parameter style
-"BARE", and mode "OUT" or "INOUT" is lifted and default rules are
-defined. This change is consistent with section 3.6.2.2 of the JAX-WS
-2.0 specification.
-
-Version 2.0
-
-* Added @WebResult.header.
-* A document "bare" style operation can have a void return type and a
-Holder as a parameter. The Holder of course would have to be INOUT. It
-could also have 2 parameters one IN and one OUT.
-* Changed @SOAPBinding to be configurable on a per operation basis rather
-than on the entire interface.
-* Made @HandlerChain.name and @SOAPMessageHandlers deprecated.
-* Added support for JAX-WS.
-* Updated section 3.1 Service Implementation Bean, Item 6 to state that
-exposing all public methods if not @WebMethod annotations are declared
-to include consideration of annotation inheritence.
-* Changed default of @WebResult.name to @WebMethod.operationName +
-"Response" for Doc/lit/bare operations.
-* Clarified @WebService.targetNamespace usage
-* Clarified @WebParam.mode usage.
-* Added @WebMethod.exclude.
-* Explicitly stated that if an implementation bean references an endpoint
-interface, it must implement all the methods in the service endpoint
-interface.
-* Changed @WebParam.name to default to arg0, arg1, etc (based on position
-in the method signature).
-* Added @WebParam.partName and @WebResult.partName to specify part name
-used in the binding.
-* Added requirement that the name for headers must be unique with an
-operation.
-* Clairified that a target namespace of "" maps to the empty namespace,
-not the web service namespace.
-* For doc/lit bare, require that any INOUT or OUT parameters must have a
-@WebParam.name specified to avoid name clashes with the input parameter.
-
-
-* Added @WebService.portName for the wsdl:port
-* Updated Handler schema
-* Added document/literal example to Using JSR-181 annotation to affect the
-shape of the WSDL
-* Included various editorial changes.
-
-Version 0.9.4
-
-* Allowing @HandlerChain and @SOAPMessageHandler on implementation when an
-endpointInterface is used.
-
-Version 0.9.3
-
-* Using RFC 2119 Keyword convention.
-* Added Retention annotation to spec annotation definitions.
-* Fixed various Java and XML syntax errors.
-* Changed Implementation Bean to expose all public method by default.
-* WSDL generation is REQUIRED.
-* Clarified support for Start with WSDL, and Start with WSDL and Java
-development modes as OPTIONAL.
-* Clarified [.underline]#mailto:@HandlerChain.file[ @HandlerChain.file]#
-attribute syntax and processing requirements.
-
-Version 0.9.2
-
-* Removed security annotations as these will be defined by JSR-250 –
-Common Annotations.
-
-Version 0.9.1
-
-* Changed default name of @WebResult to be "return" instead of "result".
-* Fixed various Java and XML syntax errors.
+. Common Annotations for the Java Platform
+https://jcp.org/en/jsr/detail?id=250
```

Comment:
this is now "Jakarta Annotations: https://jakarta.ee/specifications/annotations/2.0/ "

Reply:
ok, resolved by https://github.com/eclipse-ee4j/jws-api/pull/69/commits/0b6c04287b2400cd4ccaeb1cd387e9f5c3535f1c

Reply:
almost fine :-) all other pointers to specs do follow `name VERSION link` pattern but this one is missing the version part - can this be made consistent, please? (either remove version from other jakarta specs or add the version here)

Reply:
ok, resolved by https://github.com/eclipse-ee4j/jws-api/pull/69/commits/b03628ddda22c1bad0221dec7bfd3dc9de6788f8